articles/logging-with-winston/index.md

```diff
@@ -0,0 +1,547 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/logging-with-winston/
+title: 
+description: This article goes through 
+author: joseph-chege
+date: 2020-11-10T00:00:00-15:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/logging-with-winston/hero.png
+    alt:  example image
+---
+Assume you have an app running at the production level. If this app runs into an error mode, a million things could lead to that. Some are essential others are not. As a developer, you need to find the critical issues and solve them.
```

Comment:
**Added value edit:** Assume you have an app running in a production environment with millions of daily users that earns thousands of dollars. If there's an error, that costs money and users. No one wants to use an app that's buggy. **You could reference an example where a bug has cost a company a lot of money** There are a variety of reasons why an application could have a bug and as a developer, you need to find out why and fix it.